Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2.py

```diff
@@ -1337,12 +1396,13 @@ def main():
         args = demisto.args()
 
         if demisto.command() == "test-module":
+            demisto.debug(f"command being called is: {demisto.command()}")
```

Comment:
no need to have the same log entry under each command, i added it above to have it only once, please remove all added ones